x-pack/winlogbeat/module/security/config/winlogbeat-security.js

```diff
@@ -1095,17 +1348,31 @@ var security = (function () {
         return msobjsMessageTable[code];
     };
 
-    var addActionDesc = function(evt){
+    var addEventFields = function(evt){
         var code = evt.Get("event.code");
         if (!code) {
             return;
         }
-        var eventActionDescription = eventActionTypes[code];
+        var eventActionDescription = eventActionTypes[code][2];
         if (eventActionDescription) {
-            evt.Put("event.action", eventActionDescription);
+            evt.Put("event.category", eventActionTypes[code][0]);
+            evt.Put("event.type", eventActionTypes[code][1]);
```

Comment:
evt.AppendTo would be safer.  event.category & evet.type are arrays and in the future another function might set these before this gets run.